dbms/src/Flash/tests/gtest_spill_sort.cpp

```diff
@@ -67,6 +69,12 @@ try
     /// enable spill and use small max_cached_data_bytes_in_spiller
     context.context->setSetting("max_cached_data_bytes_in_spiller", Field(static_cast<UInt64>(total_data_size / 100)));
     ASSERT_COLUMNS_EQ_UR(ref_columns, executeStreams(request, original_max_streams));
+
+    // The implementation of topN in the pipeline model is LocalSort, and the result of using multiple threads is unstable. Therefore, a single thread is used here instead.
+    enablePipeline(true);
+    ASSERT_COLUMNS_EQ_UR(ref_columns, executeStreams(request, 1));
```

Comment:
Since the concurrency is 1, I think we can use `ASSERT_COLUMNS_EQ_R` here?

Reply:
Because of `auto ref_columns = executeStreams(request, original_max_streams);` still has to use `ASSERT_COLUMNS_EQ_UR `

Reply:
Maybe I can modify `auto ref_columns = executeStreams(request, original_max_streams);` to `auto ref_columns = executeStreams(request, 1);`

Reply:
ok, updated.
But because `UTF8MB4_GENERAL_CI` ignores case sensitivity, leading to unstable results, `ASSERT_COLUMNS_EQ_UR` is still used for `TEST_F(SpillSortTestRunner, CollatorTest)`,